ArjunaJTS/integration/src/main/java/com/arjuna/ats/jbossatx/BaseTransactionManagerDelegate.java

```diff
@@ -119,8 +122,9 @@ public void resume(final Transaction transaction)
         if (transaction == null) {
             suspend(); // This is what AtomicAction does
         } else {
+            notifyAssociationListeners(transaction, AssociatingEventType.ASSOCIATING);
```

Comment:
This needs deleting - I only added to test supporting new event types and there is no use case for listening for ASSOCIATING events.